test/unit/mock/profiles/contains-empty-file/controls/example.rb

```diff
@@ -0,0 +1,18 @@
+# copyright: 2018, The Authors
```

Comment:
Do we need a valid file in the test profile to trigger this?

Reply:
No, on master the test profile triggered the problem.

Reply:
Why does this PR include a whole profile directory when this is about the tarball?